src/current/v24.1/log-sql-statistics-to-datadog.md

```diff
@@ -117,7 +114,7 @@ SET CLUSTER SETTING sql.telemetry.query_sampling.mode = 'transaction';
 
 Configure the following [cluster settings]({% link {{ page.version.version }}/cluster-settings.md %}) to values that are dependent on the level of granularity you require and how much performance impact from frequent logging you can tolerate:
 
-- [`sql.telemetry.transaction_sampling.max_event_frequency`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-telemetry-transaction-sampling-max-event-frequency) (default `8`) is the max event frequency (events per second) at which we sample transactions for telemetry. If sampling mode is set to 'statement', this setting is ignored. In practice, this means that we only sample a transaction if 1/max_event_frequency seconds have elapsed since the last transaction was sampled.
+- [`sql.telemetry.transaction_sampling.max_event_frequency`]({% link {{ page.version.version }}/cluster-settings.md %}#setting-sql-telemetry-transaction-sampling-max-event-frequency) (default `8`) is the max event frequency (events per second) at which we sample transactions for telemetry. If sampling mode is set to `'statement'`, this setting is ignored. In practice, this means that we only sample a transaction if 1/max_event_frequency seconds have elapsed since the last transaction was sampled. Sampling impacts the volume of transaction events emitted which can have downstream impact to workload performance and third-party processing costs. Slowly increase this sampling threshold and monitor potential impact.
```

Comment:
same as above re: formatting of `max_event_frequency` inline with the prose, and duplicating the default value info here vs. cluster settings page

Reply:
suggest making first use of term 'transaction' a link to the transactions docs

Reply:
- updated formatting of `max_event_frequency`.
- keeping default values because they provide magnitude of initial setting
- first use of the term 'transaction' on the page links to the transactions docs